src/main/java/org/janelia/saalfeldlab/paintera/meshes/MeshManagerWithAssignmentForSegments.java

```diff
@@ -460,6 +457,8 @@ public void invalidateCaches()
 	{
 		Stream.of(this.blockListCacheInvalidate).forEach(InvalidateAll::invalidateAll);
 		Stream.of(this.meshCacheInvalidate).forEach(InvalidateAll::invalidateAll);
+		if (labelBlockLookup instanceof LabelBlockLookupFromN5)
```

Comment:
I'd rather add this to the `LabelBlockLookup` interface than doing an `instanceof` check for a specific implementation. May be useful to implement the entire `Invalidate` interface that imglib2-cache now provides.

Reply:
Good point, I didn't know that `Invalidate` was an interface in imglib2-cache, it's probably in one of the newer versions. Although it's a bit more granular than needed here, I'll try to make use of it. Maybe we could add another interface that would extend `LabelBlockLookup` and `Invalidate` meaning that it's a lookup with added cache functionality, and let N5-based block lookup implement it, and keep the rest as is.

Reply:
Yes, it is in a more recent version that the [remove-global-cache branch](https://github.com/saalfeldlab/paintera/pull/316) uses. Updating to that more recent version was quite some work so it might be good to way until that branch is merged.